src/main/java/teammates/ui/webapi/output/StudentData.java

```diff
@@ -19,6 +21,12 @@
     private String lastName;
     @Nullable
     private String comments;
+    @Nullable
+    private String key;
+    @Nullable
+    private String institute;
+    @Nullable
+    private Map<String, String> courseIdToInstructorGoogleIdMap;
```

Comment:
Why do we need `institute ` and `courseIdToInstructorGoogleIdMap`?

Again, we have already violated the principle that `StudentData.java` should only contains information related to a student. `institute` is technically yes but definitely not `courseIdToInstructorGoogleIdMap`/

Reply:
Yea my bad, we could just make a call on front end side for all courses. For institute however, I could not find any endpoint through which I could get institute, so i included it here. wdyt?

Reply:
Why do we need `institute`? The currently implementation is a bit expensive especially for the for each loop.

Reply:
I included it to match the current object being returned by the Admin endpoint, which has `institute` in it

Reply:
I think you should not consider "what the old endpoint will return". Rather, you should try to answer "whether I need the data or not".

Reply:
Cool, that actually makes things easier for the search service 😆

Reply:
Hmm, sorry, will include `institute` because it is being used on `admin-search-page`